lib/saltbase.pm

```diff
@@ -100,6 +100,10 @@ sub logs_from_salt {
             record_soft_failure('bsc#1211591');
             return;
         }
+        if (script_run('grep "Encountered StreamClosedException" /var/log/salt/master') == 0) {
+            record_soft_failure('bsc#1213635');
+            return;
+        }
```

Comment:
`    if (script_run("$error") != 1) {
        my $flag = 0;
        if (check_var('HOSTNAME', 'master') && script_run('grep "self.pusher.connect(timeout=timeout)" /var/log/salt/master') == 0) {
            record_soft_failure('bsc#1209248');
            $flag = 1;
        }
        if (script_run('grep "ModuleNotFoundError.*\'salt.ext.six\'" /var/log/salt/minion') == 0) {
            record_soft_failure('bsc#1211591');
            $flag = 1;
        }
        if (script_run('grep "Encountered StreamClosedException" /var/log/salt/master') == 0) {
            record_soft_failure('bsc#1213635');
            $flag = 1;
        }
        return if $flag;`

Reply:
I think in this way every error log can be recorded if there